src/java.desktop/share/classes/javax/swing/border/EtchedBorder.java

```diff
@@ -150,59 +149,23 @@ private void paintBorderShadow(Graphics g, Color c, int w, int h, int stkWidth)
      * @param height the height of the painted border
      */
     public void paintBorder(Component c, Graphics g, int x, int y, int width, int height) {
-        // We remove any initial transforms to prevent rounding errors
-        // when drawing in non-integer scales
-        AffineTransform at = null;
-        Stroke oldStk = null;
-        int stkWidth = 1;
-        boolean resetTransform = false;
+        SwingUtilities3.paintBorder(c, g, x, y, width, height, this::paintUnscaledBorder);
+    }
+
+    private void paintUnscaledBorder(Component c, Graphics g, int x, int y,
+                                    int w, int h, double scaleFactor) {
```

Comment:
```suggestion
    private void paintUnscaledBorder(Component c, Graphics g, int x, int y,
                                     int w, int h, double scaleFactor) {
```
One more space is missing to align.